data/operators/amenity/car_sharing.json

```diff
@@ -463,6 +463,7 @@
       "displayName": "Régie des Transports du Territoire de Belfort",
       "id": "regiedestransportsduterritoiredebelfort-672a50",
       "locationSet": {"include": ["fx"]},
+      "matchNames": ["rttb"],
```

Comment:
Should this entry have `operator:short` set to `RTTB`? If so, the `matchNames` will be redundant because NSI will get that from `operator:short`.

Reply:
moved `RTTB` to `operator:short` in next commit